quickjs.c

```diff
@@ -38057,10 +38057,10 @@ static JSValue js_error_set_stackTraceLimit(JSContext *ctx, JSValue this_val, JS
 {
     if (JS_IsUndefined(this_val) || JS_IsNull(this_val))
         return JS_ThrowTypeErrorNotAnObject(ctx);
-    int limit;
-    if (JS_ToInt32(ctx, &limit, value) < 0)
+    double limit = 0;
+    if (JS_ToFloat64(ctx, &limit, value) < 0)
         return JS_EXCEPTION;
-    ctx->error_stack_trace_limit = limit;
+    ctx->error_stack_trace_limit = isinf(limit) ? 2147483647 : limit;
```

Comment:
Why not INT32_MAX?
```suggestion
    ctx->error_stack_trace_limit = isinf(limit) ? INT32_MAX : limit;
```
But you changed the conversion from JS_ToInt32 to JS_ToFloat64 and that means NaNs can also show up now.

Casting NaN to an integer is UB. The logic should probably be something like:
```suggestion
    ctx->error_stack_trace_limit = isfinite(limit) ? limit : INT32_MAX;
```

Reply:
I tried using `INT_MAX` but it was missing on Linux.

Reply:
It's INT32_MAX